joomla/joomla-cms
# [5.0] Adding display of progress to installation

### Summary of Changes
The current installation is rather opaque when it comes to doing the actual setup on the server. There is no real progress indicator and it also isn't clear what Joomla is doing during installation. I assume that is also the reason why some autoinstallers just fill out the configuration.php and execute our SQL scripts, which simply isn't enough to properly install Joomla.
This PR introduces an additional step in the GUI, replacing the joomla-core-loader element and instead it displays a progress bar ~~and the different tasks the installer has so far executed~~.

### Testing Instructions
Simply apply the PR and execute the installation. See that when you click on the "Install Joomla" button, you get a new screen with progress bar ~~and a list of steps Joomla is doing~~.

### Documentation Changes Required
None.
